app/models/user.rb

```diff
@@ -1,4 +1,11 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Kindly enable `:confirmable` to make it possible to get a confirmation after login 👍